source/beamline/AMPVControl.cpp

```diff
@@ -50,6 +56,17 @@ AMReadOnlyPVControl::AMReadOnlyPVControl(const QString& name, const QString& rea
 
 }
 
+
+void AMReadOnlyPVControl::enableLimitMonitoring()
```

Comment:
If the PV does not have a HOPR LOPR (for read-only PVS) or a DRVH DRVL (for writable PVS) then there is nothing to monitor, and attempting to connect to these will result in the usual ``` AMProcessVariable: channel connect timed out for blah blah blah``` messages.

For example all the enum PV types (mbbi records) don't support anything but HOPR.